Parsers/ASimNetworkSession/Parsers/vimNetworkSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,362 @@
+          DstUsername = DestinationUserName,
+          DvcId = DeviceExternalID,
+          DvcOriginalAction = DeviceAction,
+          EventOriginalSeverity = LogSeverity,
+          DstZone = DeviceCustomString5,
+          EventOriginalType = DeviceEventClassID,
+          EventOriginalUid = ExtID,
+          EventProductVersion = DeviceVersion,
+          NetworkPackets = FieldDeviceCustomNumber2,
+          NetworkRuleName = DeviceCustomString1,
+          SrcDvcId = PanOSSourceUUID,
+          SrcGeoCountry = PanOSSourceLocation,
+          SrcMacAddr = PanOSSourceDeviceMac,
+          SrcNatIpAddr = SourceTranslatedAddress,
+          SrcNatPortNumber = SourceTranslatedPort,
+          SrcPackets = PanOSPacketsSent,
+          SrcPortNumber = SourcePort,
+          SrcUsername = SourceUserName,
+          SrcZone = DeviceCustomString4,
+          DvcScopeId = PanOSCortexDataLakeTenantID,
+          EventOriginalSubType = Activity,
+          EventOriginalResultDetails = Reason,
+          SrcUserId = SourceUserID,
+          DstUserId = DestinationUserID,
+          DvcInboundInterface = DeviceInboundInterface,
+          DvcOutboundInterface = DeviceOutboundInterface,
+          SrcAppName = ApplicationProtocol,
+          ThreatOriginalRiskLevel = PanOSApplicationRisk
+      | extend
+          Dvc = coalesce(DvcFQDN, DvcId, DvcHostname, DvcIpAddr),
+          EventEndTime = EventStartTime,
+          Dst = coalesce(DstDvcId, DstHostname, DstIpAddr),
+          Src = coalesce(SrcDvcId, SrcHostname, SrcIpAddr),
+          DstUserType = _ASIM_GetUserType(DstUsername, ""),
+          NetworkProtocol = toupper(Protocol),
+          NetworkBytes = SrcBytes + DstBytes,
+          NetworkProtocolVersion = case(
+                              DstIpAddr contains ".",
+                              "IPv4",       
+                              DstIpAddr contains ":",
+                              "IPv6", 
+                              ""
+                          ),
+          NetworkDirection = iff(PanOSIsClienttoServer == "true", "Outbound", "Inbound"),
+          Rule = NetworkRuleName,
+          SrcUserType = _ASIM_GetUserType(SrcUsername, ""),
+          DstUsernameType = _ASIM_GetUsernameType(DstUsername),
+          DvcIdType = iff(isnotempty(DvcId), "Other", ""),
+          SrcUsernameType = _ASIM_GetUsernameType(SrcUsername),
+          Duration = NetworkDuration,
+          IpAddr = SrcIpAddr,
+          SessionId = NetworkSessionId,
+          User = DstUsername,
+          Hostname = DstHostname,
+          SrcDvcIdType = iff(isnotempty(SrcDvcId), "Other", ""),
+          DstDvcIdType = iff(isnotempty(DstDvcId), "Other", ""),
+          SrcDomainType = iff(isnotempty(SourceNTDomain), "Windows", SrcDomainType),
+          DstDomainType = iff(isnotempty(DestinationNTDomain), "Windows", DstDomainType),
+          SrcUserIdType = iff(isnotempty(SrcUserId), "UID", ""),
+          DstUserIdType = iff(isnotempty(DstUserId), "UID", ""),
```

Comment:
@niralishah-crest 
is this linux user id?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/38cf5867-6d17-4247-84df-41eebd6e7c78)